docs/design/autoware-architecture/sensing/data-types/radar-data/radar-pointcloud-data.md

```diff
@@ -0,0 +1,75 @@
+# Radar pointcloud data pre-processing design
+
+## Overview
+
+### Pipeline
+
+This diagram describes the pre-process pipeline for radar pointcloud.
+
+![radar-pointcloud-sensing](image/radar-pointcloud-sensing.drawio.svg)
+
+### Interface
+
+- Input
+  - Radar data from device
+  - Twist information of ego vehicle motion
+- Output
+  - Dynamic radar pointcloud (`ros-perception/radar_msgs/msg/RadarScan.msg`)
```

Comment:
What is Dynamic radar pointcloud?

Reply:
It means that pointcloud which is not static.
To express brevity for interface documentation and I think who read this document for the end can understand this word, so I would not like to change it.